diff --git a/dict/Ngu-vung-Danh-tu-Thien-hoc/Ngu-vung-Danh-tu-Thien-hoc.opf b/dict/Ngu-vung-Danh-tu-Thien-hoc/Ngu-vung-Danh-tu-Thien-hoc.opf
index addd0be..8bdb4ad 100644
--- a/dict/Ngu-vung-Danh-tu-Thien-hoc/Ngu-vung-Danh-tu-Thien-hoc.opf
+++ b/dict/Ngu-vung-Danh-tu-Thien-hoc/Ngu-vung-Danh-tu-Thien-hoc.opf
@@ -9,8 +9,10 @@
 	<dc-metadata>
 		<dc:Identifier id="uid">Ngu-vung-Danh-tu-Thien-hoc</dc:Identifier>
 		<!-- Title of the document -->
-		<dc:Title><h2>Ngu-vung-Danh-tu-Thien-hoc</h2></dc:Title>
-		<dc:Language>EN</dc:Language>
+		<dc:Title><h2>Ngữ vựng danh từ Thiền học v1.0</h2></dc:Title>
+		<dc:Language>vi</dc:Language>
+		<dc:publisher>catus.phan@gmail.com</dc:publisher>
+		<dc:Creator>catus.phan@gmail.com</dc:Creator>
 	</dc-metadata>
 	<x-metadata>
 	        <output encoding="utf-8" flatten-dynamic-dir="yes"/>
