diff --git a/dict/TudienThienChuu/TudienThienChuu.opf b/dict/TudienThienChuu/TudienThienChuu.opf
index 168f9c7..da7c17b 100644
--- a/dict/TudienThienChuu/TudienThienChuu.opf
+++ b/dict/TudienThienChuu/TudienThienChuu.opf
@@ -9,8 +9,11 @@
 	<dc-metadata>
 		<dc:Identifier id="uid">TudienThienChuu</dc:Identifier>
 		<!-- Title of the document -->
-		<dc:Title><h2>TudienThienChuu</h2></dc:Title>
-		<dc:Language>EN</dc:Language>
+		<dc:Title><h2>Từ điển Hán Việt Thiền Chửu v1.3</h2></dc:Title>
+		<dc:Creator>catusf</dc:Creator>
+		<dc:Language>vi</dc:Language>
+		<dc:publisher>catus.phan@gmail.com</dc:publisher>
+		<dc:Creator>catus.phan@gmail.com</dc:Creator>
 	</dc-metadata>
 	<x-metadata>
 	        <output encoding="utf-8" flatten-dynamic-dir="yes"/>
